source/Jobbr.Server.UnitTests/GlobalSuppressions.cs

```diff
@@ -0,0 +1,8 @@
+﻿// This file is used by Code Analysis to maintain SuppressMessage
+// attributes that are applied to this project.
+// Project-level suppressions either have no target or are given
+// a specific target and scoped to a namespace, type, member, etc.
+
+using System.Diagnostics.CodeAnalysis;
+
+[assembly: SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "No documentation needed for test classes")]
```

Comment:
Ah I think this is better done, by duplicating the ruleset file for tests, including it in the project and changing the rule there.

https://github.com/jobbrIO/jobbr-server/blob/develop/source/JobbrRuleSet.ruleset